libbfio/src/handle.rs

```diff
@@ -45,68 +48,119 @@ impl Handle {
     }
 }
 
-type io_handle_t = *mut IoHandle;
+pub type IoHandleConstRef = *const IoHandle;
+pub type IoHandleRefMut = *mut IoHandle;
+pub type BoxedIoHandleRefMut = *mut *mut IoHandle;
+
+//enum LibbfioFlags {
+//    /* The IO handle is not managed by the library
+//     */
+//    IoHandleNonManaged = 0x00,
+//
+//    /* The IO handle is managed by the library
+//     */
+//    IoHandleManaged = 0x01,
+//
+//    /* The IO handle is cloned by the function
+//     */
+//    CloneByFunction = 0x00,
+//
+//    /* The IO handle is not cloned, but passed as a reference
+//     */
+//    CloneByReference = 0x02,
+//}
+
+/* The access flags definitions
+ * bit 1						set to 1 for read access
+ * bit 2						set to 1 for write access
+ * bit 3						set to 1 to truncate an existing file on write
+ * bit 4-8						not used
+ */
+pub enum LibbfioAccessFlags {
```

Comment:
Nice. Doesn't bindgen generate anything that you can use for the value part?

Reply:
it was a bit annoying, but I've used the imported types